ui/app/helpers/mountable-secret-engines.js

```diff
@@ -134,6 +134,9 @@ const MOUNTABLE_SECRET_ENGINES = [
   },
 ];
 
+// A list of Workflow Identity Federation engines. Will eventually include Azure and GCP.
+export const wifEngines = ['aws'];
```

Comment:
Thanks for the comment here! It's helpful to know what's coming. Since this is a const (and not a function) I think our pattern is to make this all caps `WIF_ENGINES`. Will this will be needed in a template at some point? If so, then we should probably do something similar to `MOUNTABLE_SECRET_ENGINES` and make this a function, though that doesn't seem to be necessary here
```js
export const WIF_ENGINES = ['aws'];
export function wifEngines() {
  return WIF_ENGINES.slice();
}
```